src/parse/asp/lexer.go

```diff
@@ -85,13 +85,17 @@ func newLexer(r io.Reader) *lex {
 
 // A lex is a lexer for a single BUILD file.
 type lex struct {
-	b      []byte
-	i      int
-	line   int
-	col    int
+	// The raw bytes we're lexing
+	bytes []byte
```

Comment:
Might be nice to make this a reader, and read bytes off it at a time to avoid allocations